feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsScreen.kt

```diff
@@ -119,29 +127,41 @@ private fun InterestsContent(
     modifier: Modifier = Modifier
 ) {
     Column(modifier) {
-        NiaTabRow(selectedTabIndex = tabState.currentIndex) {
+        val pagerState = rememberPagerState()
+        val scope = rememberCoroutineScope()
+        NiaTabRow(selectedTabIndex = pagerState.currentPage) {
             tabState.titles.forEachIndexed { index, titleId ->
                 NiaTab(
-                    selected = index == tabState.currentIndex,
-                    onClick = { switchTab(index) },
+                    selected = pagerState.currentPage == index,
+                    onClick = {
+                        switchTab(index)
+                        scope.launch {
+                            pagerState.scrollToPage(index)
+                        }
+                    },
                     text = { Text(text = stringResource(id = titleId)) }
                 )
             }
         }
-        when (tabState.currentIndex) {
-            0 -> {
-                TopicsTabContent(
+        HorizontalPager(
+            state = pagerState,
+            count = tabState.titles.size,
+            verticalAlignment = Alignment.Top,
+            contentPadding = PaddingValues(top = 12.dp),
+        ) { index ->
```

Comment:
I believe that the UI test `interestsWithTopics_whenAuthorsFollowed_showFollowedAndUnfollowedTopicsWithInfo` is failing because of a bug here which is preventing the "People" tab from being displayed. 

Context: https://github.com/android/nowinandroid/actions/runs/3447249174/jobs/5753772088

Reply:
The test is failing for me locally so it's definitely not flaky CI. 

To run this test locally do `./gradlew :feature:interests:connectedDemoDebugAndroidTest`.